corehq/apps/data_interfaces/models.py

```diff
@@ -12,6 +14,15 @@
 ALLOWED_DATE_REGEX = re.compile('^\d{4}-\d{2}-\d{2}')
 AUTO_UPDATE_XMLNS = 'http://commcarehq.org/hq_case_update_rule'
 
+class PropertyTypeChoices(object):
+    EXACT = "Exact"
+    CASE_PROPERTY = "Case Property"
```

Comment:
i generally prefer these constants to be code-like, for example, `EXACT` and `CASE_PROPERTY`